timed/tracking/models.py

```diff
@@ -103,6 +103,19 @@ class Report(models.Model):
     rejected = models.BooleanField(default=False)
     remaining_effort = models.DurationField(default=timedelta(0), null=True)
 
+    class Meta:
+        """Meta information for the report model."""
+
+        indexes = (models.Index(fields=["date"]),)
+
+    def __str__(self):
+        """Represent the model as a string.
+
+        :return: The string representation
+        :rtype:  str
```

Comment:
That docstring style could be cleaned up while we're at it. sphinx-doc I think used this format, but we don't do that anymore

Reply:
i moved all `__str__`s and some others (clean, validate on serializers, worktime_per_day, duration etc,) to type annotations here: https://github.com/adfinis/timed-backend/pull/1049/commits/78cd741caf2add4c1bde1c12065a5d0a2ae445d2

The rest i would like to do when adding types (https://github.com/adfinis/timed-backend/issues/1054#issuecomment-2068946433)